src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEModuleSymbol.cs

```diff
@@ -406,12 +405,18 @@ internal TypeSymbol TryDecodeAttributeWithTypeArgument(EntityHandle handle, Attr
         /// <param name="token"></param>
         /// <param name="foundExtension">True if we found an extension method, false otherwise.</param>
         /// <returns>The attributes on the token, minus any ExtensionAttributes.</returns>
-        internal ImmutableArray<CSharpAttributeData> GetCustomAttributesFilterExtensions(EntityHandle token, out bool foundExtension)
+        internal ImmutableArray<CSharpAttributeData> GetCustomAttributesFilterCompilerAttributes(EntityHandle token, out bool foundExtension, out bool foundReadOnly)
         {
-            CustomAttributeHandle extensionAttribute;
-            var result = GetCustomAttributesForToken(token, out extensionAttribute, AttributeDescription.CaseSensitiveExtensionAttribute);
+            var result = GetCustomAttributesForToken(
+                token,
+                out CustomAttributeHandle extensionAttribute,
+                AttributeDescription.CaseSensitiveExtensionAttribute,
+                out CustomAttributeHandle isReadOnlyAttribute,
+                AttributeDescription.IsReadOnlyAttribute,
+                out _, default, out _, default);
```

Comment:
>out _, default, out _, default [](start = 16, length = 30)

nit: consider naming those arguments

Reply:
Do you mean adding argument labels or renaming the `_` to something else (and if so, what would they be named?)

Reply:
I mean naming the arguments using parameter names: `parameterX: out _, parameterY: default, ...`

Reply:
Done